substratest/assets.py

```diff
@@ -130,8 +130,7 @@ class _ComputePlanFutureMixin(_BaseFutureMixin):
 
 
 _MANUAL_CONVERTION_FIELDS = {
-    'authorizedIDs': 'authorized_ids',
-    'dataManagerKey': 'dataset_key',
+    'data_manager_key': 'dataset_key',
```

Comment:
@jmorel Do you remember why we convert data manager to dataset ?

Reply:
I thought I already posted a reply to this, but it's not in this PR, so here it is again :)

The idea is that data_manager is an old name that we try to phase out in favor of dataset. This allows the tests developers to not think about the old name and only think in terms of dataset.